icloudpd/base.py

```diff
@@ -350,17 +350,17 @@ def photos_exception_handler(ex, retries):
     # pylint: disable-msg=too-many-nested-blocks
     for photo in photos_enumerator:
         for _ in range(constants.MAX_RETRIES):
-            if skip_videos and photo.item_type != "image":
-                logger.set_tqdm_description(
-                    "Skipping %s, only downloading photos." % photo.filename
-                )
-                break
-            if photo.item_type != "image" and photo.item_type != "movie":
-                logger.set_tqdm_description(
-                    "Skipping %s, only downloading photos and videos. "
-                    "(Item type was: %s)" % (photo.filename, photo.item_type)
-                )
-                break
+            # if skip_videos and photo.item_type != "image":
```

Comment:
If this is really causing errors, can you provide some stacktrace or insight of what's breaking right now? Also, this should be a separate pull request and remove the code completely. Commenting out broken out will give people the impression that it could work, when in fact it's broken...